widget-context.php

```diff
@@ -418,16 +418,10 @@ function context_check_url( $check, $settings ) {
 
 	function match_path( $patterns ) {
 
-		global $wp;
-
 		$patterns_safe = array();
 
-		// Get the request URI from WP
-		$url_request = $wp->request;
-
-		// Append the query string
-		if ( ! empty( $_SERVER['QUERY_STRING'] ) )
-			$url_request .= '?' . $_SERVER['QUERY_STRING'];
+		// Get the request URI
+		$url_request = trim( $_SERVER['REQUEST_URI'], '/' );

```

Comment:
@lachmanski So with the WP in a subfolder we're expecting users to enter path rules relative to the root of the domain, correct?